### PR TITLE
Add ScottoFly Keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
Add ScottoFly handwired keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/scottokeebs/scottofly/info.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        "rows": ["B1", "B3", "B2", "B6"]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x10_6": {
```
Comment (dunk2k) [inline keyboards/handwired/scottokeebs/scottofly/info.json:29]: Utterly optional, but changing `LAYOUT_ortho_3x10_6` to `LAYOUT_split_3x5_3` will allow community layouts to automatically be assigned (namely miryoku).
Would have to be changed in all `keymap.c` files also.

File: keyboards/handwired/scottokeebs/scottofly/info.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        //  3, 4, 5, 6, 7, 8, 9, A3, A2, A1
+        "cols": ["D0", "D4", "C6", "D7", "E6", "B4", "B5", "F4", "F5", "F6"]
```
[the remark this answers is not in the document]
Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scottofly/info.json]: Good catch, fixed in [baca888](https://github.com/qmk/qmk_firmware/pull/20939/commits/baca888c7185bd8958196aab1944f04986f04c1e)

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/scottokeebs/scottofly/readme.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# Compiling
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/jscotto/scottofly:default
+
+Flashing example for this keyboard:
+
+    make handwired/jscotto/scottofly:default
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottofly/readme.md:19]:
```suggestion
    make handwired/scottokeebs/scottofly:default

Flashing example for this keyboard:

    make handwired/scottokeebs/scottofly:default
```

File: keyboards/handwired/scottokeebs/scottofly/readme.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# ScottoFly
+
+![ScottoFly](https://i.imgur.com/TXuNxt7h.jpg)
+
+A 36 key column staggered ergonomic keyboard. Case files available [here](https://github.com/joe-scotto/scottokeebs).
+
+-   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
+-   Hardware Supported: ATmega32U4
+-   Hardware Availability: [Amazon](https://amazon.com)
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottofly/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
* Hardware Supported: ATmega32U4
* Hardware Availability: [Amazon](https://amazon.com)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Please revert the submodule changes.

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**